composer.json

```diff
@@ -77,5 +77,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
```

Comment:
opening pandora box....

makefile?

Reply:
Makefile would (nomen omen) make sense if we needed to use non-PHP scripts, that are not PHP-ready. When we have Composer scripts, we can run them the same way in the Docker container (as I said, only `git` would be needed). Makefile could add shortcuts for running those scripts on Docker, though, but it's separate feature for me.

Reply:
Why not [`act`](https://github.com/nektos/act)?

Reply:
I've never used `act` but as far as I see it's not what I would like to use. There is no need to imitate whole Github Actions since we have Docker runtime ready to use. From my experience it's better to provide common scripts like Composer ones and just run them locally *and* in the CI. Those processes should be built from bottom to top - small commands as common entrypoints, then runtimes on top of them.

Besides, I did not read deeply `act`'s docs, but would it run locally whole Github Actions' suite? That's not something I would like to see 😅.

Reply:
I really like composer scripts and also use them when possible. IMO having them in one place and don't need this project specific dev-tools dir is a big plus.